### PR TITLE
pbmtolj, pjtoppm, pnmtopclxl, ppmtolj, ppmtopj, ppmtospu, sldtoppm, spctoppm, sputoppm, thinkjettopbm: add pages

### PR DESCRIPTION
**Pull request by gutjuri**
- [x] The page(s) are in the correct platform directories: `common`, `linux`, `osx`, `windows`, `sunos`, `android`, etc.
- [x] The page(s) have at most 8 examples.
- [x] The page description(s) have links to documentation or a homepage.
- [x] The page(s) follow the [content guidelines](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#guidelines).
- [x] The PR title conforms to the recommended [templates](/tldr-pages/tldr/blob/main/CONTRIBUTING.md#commit-message).
- **Version of the command being documented (if known):**

### CODE REVIEW

**Comment by gutjuri** [comment]
> Please double-check your description and the first command description. These two are sometimes the same, but sometimes they are not. Try to be as consistent as possible to create a standard or template for Netpbm pages.

Yes, I try to make them different whenever possible (e.g. writing out the name of a format in the command description but not in the example descriptions).

File: pages/common/pbmtolj.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# pbmtolj
+
+> Convert a PBM file to an HP LaserJet file.
```
[the remark this answers is not in the document]
Comment (gutjuri) [reply pages/common/pbmtolj.md:3]: This a a correct "an".

File: pages/common/pbmtolj.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://netpbm.sourceforge.net/doc/pbmtolj.html>.
+
+- Convert a PBM file to an HP LaserJet file:
```
[the remark this answers is not in the document]
Comment (gutjuri) [reply pages/common/pbmtolj.md:6]: This a a correct "an".

File: pages/common/pnmtopclxl.md [hunk 3]
```diff
@@ -0,0 +1,16 @@
+# pnmtopclxl
+
+> Convert a PNM file to an HP LaserJet PCL XL printer stream.
```
[the remark this answers is not in the document]
Comment (gutjuri) [reply pages/common/pnmtopclxl.md:3]: This a a correct "an".

File: pages/common/pnmtopclxl.md [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://netpbm.sourceforge.net/doc/pnmtopclxl.html>.
+
+- Convert PNM files to an HP LaserJet PCL XL printer stream:
```
[the remark this answers is not in the document]
Comment (gutjuri) [reply pages/common/pnmtopclxl.md:6]: This a correct "an".

File: pages/common/ppmtolj.md [hunk 5]
```diff
@@ -0,0 +1,16 @@
+# ppmtolj
+
+> Convert a PPM file to an HP LaserJet PCL 5 Color file.
```
[the remark this answers is not in the document]
Comment (gutjuri) [reply pages/common/ppmtolj.md:3]: This a correct "an".

File: pages/common/ppmtolj.md [hunk 6]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://netpbm.sourceforge.net/doc/ppmtolj.html>.
+
+- Convert a PPM file to an HP LaserJet PCL 5 Color file:
```
[the remark this answers is not in the document]
Comment (gutjuri) [reply pages/common/ppmtolj.md:6]: This a correct "an".

File: pages/common/ppmtopj.md [hunk 7]
```diff
@@ -0,0 +1,16 @@
+# ppmtopj
+
+> Convert a PPM file to an HP PaintJet file.
```
[the remark this answers is not in the document]
Comment (gutjuri) [reply pages/common/ppmtopj.md:3]: This a correct "an".

File: pages/common/ppmtopj.md [hunk 8]
```diff
@@ -0,0 +1,16 @@
...
+> More information: <https://netpbm.sourceforge.net/doc/ppmtopj.html>.
+
+- Convert a PPM file to an HP PaintJet file:
```
[the remark this answers is not in the document]
Comment (gutjuri) [reply pages/common/ppmtopj.md:6]: This a correct "an".

File: pages/common/pnmtopclxl.md [hunk 9]
```diff
@@ -0,0 +1,16 @@
...
+
+`pnmtopclxl {{path/to/input1.pnm path/to/input2.pnm ...}} > {{path/to/output.pclxl}}`
+
+- Specify the resolution of the image aswell as the location of the page from the upper left corner of each image:
```
Comment (kbdharun) [inline pages/common/pnmtopclxl.md:10]:
```suggestion
- Specify the resolution of the image as well as the location of the page from the upper left corner of each image:
```

File: pages/common/ppmtopj.md [hunk 10]
```diff
@@ -0,0 +1,16 @@
...
+
+`ppmtopj {{path/to/input.ppm}} > {{path/to/output.pj}}`
+
+- Move the image in the x- and y-direction:
```
Comment (kbdharun) [inline pages/common/ppmtopj.md:10]:
```suggestion
- Move the image in the x and y direction:
```

**Review by kbdharun — APPROVED** [review]
LGTM, Thanks for your contribution.

**Pull request merged**